expo/snack
# [snack-sdk] Upgrade to Expo SDK 41

# Why

Upgrade to support Expo SDK 41

# How

Followed steps [described here](https://github.com/expo/snack/pull/109/files#diff-5172fce9dfcb48632737b8249989e0d298304200575759ea04ced389284da81eR41).

# Test Plan

Staging should be updated with the changes required for SDK 41.

- Test snack.expo.io (with no id!)
  - Open unsaved snack in app, make some changes, then hit save
- Test existing snack.expo.io/ID (https://snack.expo.io/r1Xun_7eb)
  - Open that in the app, make some changes, save, and reload
- Test existing permanent snack snack.expo.io/@username/project
  - Open that in the app, make some changes, save, and reload
- Open snack directly on phone without website open (https://expo.io/@snack/r1Xun_7eb)
- Open permanent snack directly on phone without website open (https://expo.io/@username/project)
- Open `website/snack-embed.html` to test embeds
